docs/pages/component/events.md

```diff
@@ -493,7 +493,24 @@ Example:
 }
 ```
 
-Platforms: Android
+### `onSubtitleTracks`
+Callback function that is called when subtitles are available. This provides the actual subtitle content.
+
+Payload:
+
+Property | Type | Description
+--- | --- | ---
+`subtitleTracks` | `string` | The subtitles text content in a compatible format.
+
+
+Example:
+```javascript
+{
+  subtitleTracks: "This blade has a dark past.",
```

Comment:
do it make sens to add a duration ? not sure about ios api & webVtt format but in terms of api it would make sense (not mandatory to merge this PR)

Reply:
Yes it does, I can take a look at this and add it as another property on another PR 😊